src/validators/eventValidator.ts

```
@@ -0,0 +1,26 @@
+    throw new ValidationError("Event date must be in the future");
+  }
+  if (!data.location?.trim()) {
+    throw new ValidationError("Location is required");
+  }
+  if (!data.capacity || data.capacity < 1) {
+    throw new ValidationError("Capacity must be at least 1");
+  }
+  if (!data.price || data.price < 0) {
+    throw new ValidationError("Price cannot be negative");
+  }
+};
```

ESLint

[error] Insert `⏎`

(prettier/prettier)